bindata/network/ovn-kubernetes/common/001-crd.yaml

```diff
@@ -713,14 +720,14 @@ spec:
                 items:
                   type: string
                 type: array
+                x-kubernetes-list-type: set
               status:
                 description: A concise indication of whether the AdminPolicyBasedRoute
                   resource is applied with success
                 type: string
             required:
             - lastTransitionTime
             - messages
-            - status
```

Comment:
I talked with @npinaeva about this being a backward incompatible change. Need to confirm if we are ok with this.